htlcswitch/interceptable_switch.go

```diff
@@ -486,6 +486,21 @@ func (s *InterceptableSwitch) interceptForward(packet *htlcPacket,
 			return false, nil
 		}
 
+		var customRecordsBlob []byte
+
+		htlc.CustomRecordsBlob.WhenSome(
+			func(b tlv.RecordT[lnwire.CustomRecordsBlobTlvType,
+				[]byte]) {
+
+				customRecordsBlob = b.Val
+			},
+		)
+
+		customPeerRecords := new(record.CustomSet)
+		customPeerRecords.Decode(bytes.NewReader(customRecordsBlob))
```

Comment:
Should propagate the decode error. IIUC, that can fail if the incoming wire records aren't canonical for w/e reason.

Reply:
this diff changed, and we don't decode on this level

closing comment